CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ log is based on the [Keep a CHANGELOG](http://keepachangelog.com/) project.
 - Firmware metrics and request headers update for Dell iDRAC9 with FW ver.3.xx and 4.xx [#77](https://github.com/Comcast/fishymetrics/issues/77)
 - Power supply status duplicate bay number metrics [#85](https://github.com/Comcast/fishymetrics/issues/85)
 - Capturing Model field in finished scrape log message [#94](https://github.com/Comcast/fishymetrics/issues/94)
+- Removed trailing spaces in label values [#96](https://github.com/Comcast/fishymetrics/issues/96)
```

Comment:
Please pull in latest CHANGELOG, I've added the v0.12.0 release and started a new Unreleased section

Reply:
Addressed in commit [0c3e1a0](https://github.com/Comcast/fishymetrics/pull/99/commits/0c3e1a0b361d3a037a02f4213b2343bd8f35edef)